crates/core/src/rpc/minimal.rs

```diff
@@ -668,12 +668,40 @@ impl Minimal for SurfpoolMinimalRpc {
     fn get_block_height(
         &self,
         meta: Self::Metadata,
-        _config: Option<RpcContextConfig>,
+        config: Option<RpcContextConfig>,
     ) -> Result<u64> {
+        let config = config.unwrap_or_default();
+
+        //get the current slot and check min_context_slot constraint
```

Comment:
```suggestion
```
A good practice for comments that I've been taught by my seniors is:
 - DO write comments if it explains _why_ we're doing something, and it's not immediately clear
 - Don't write comments if it's explaining _what_ you're doing - this should be clear from the code